docs/server/source/production-deployment-template/node-on-kubernetes.rst

```diff
@@ -500,6 +500,19 @@ Step 12: Start a Kubernetes StatefulSet for MongoDB
     backend port. Set it to the value specified for ``mongodb-backend-port``
     in the ConfigMap.
 
+  * **Optional**: You can also change the value for ``STORAGE_ENGINE_CACHE_SIZE``, for more information
+    regarding this configuration, please consult the `MongoDB Official
+    Documentation <https://docs.mongodb.com/manual/reference/configuration-options/#storage.wiredTiger.engineConfig.cacheSizeGB>`_.
+
+  * **Optional**: If you are not using the **Standard_D2_v2** virtual machines for Kubernetes agents as per the guide,
```

Comment:
generally recommended specs would be better instead of azure specific instance type

Reply:
The general specs are down below #L513 and #L514. Referring to Azure specific size here because the template uses that.